kits/multisig/contracts/MultisigKit.sol

```diff
@@ -59,8 +61,12 @@ contract MultisigKit is BetaKitBase {
             1825 days // ~5 years
         );
 
-        // support modification permission
         ACL acl = ACL(Kernel(voting.kernel()).acl());
+
+        // create vote permission
+        acl.createPermission(tokenManager, voting, voting.CREATE_VOTES_ROLE(), voting);
```

Comment:
Just a note: We need to modify the radspec string for the voting forward function, as when the token manager forwards to the voting app the vote is not casted (as discussed offline)

<img width="386" alt="2018-09-27 at 9 53 18 pm" src="https://user-images.githubusercontent.com/447328/46171609-4b152400-c2a1-11e8-9e61-9a04bd026d05.png">

Reply:
Yes, but how should we change it? the thing is that vote is going to be cast if `msg.sender` is a token holder, in this case sender is Token Manager, so that doesn't hold (and even if it did, it would be confusing as it wouldn't be the original sender's vote), but how do we evaluate that in a radspec string?

Reply:
For now I think we can change the text to say "casts a support vote if possible" and in the user documentation state that it's not possible in the event the create vote is forwarded by someone else.

Later on, we can add radspec helpers to detect this scenario (or improve this flow altogether somehow).

See https://github.com/aragon/aragon-apps/pull/470.